### PR TITLE
should add functionality to add comments to an article - task 8

### PR DESCRIPTION
**Pull request by Jay7735**

### CODE REVIEW

**Review by Augs0 — CHANGES_REQUESTED** [review]
This is looking great Jay well done! One tiny thing - could you hardcode the user so that comments can only come from someone 'logged in'? After that, I'm happy to approve!

File: fe-nc-news/src/AddComment.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import {React, useState, useEffect} from 'react'
+import { postComment } from "./utils/ApiFunctions";
+import "./styles/articles.css"
+
+const AddComment = ({article_id}) => {
```
Comment (MRombi) [inline fe-nc-news/src/AddComment.jsx]: We might want to pass a state that allows us to update all the comments, at the moment when we write two new comments, the first one we wrote will not be rendered 🙂

File: fe-nc-news/src/AddComment.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    })
+}, [commentAuthor, commentBody])
+
+
+const handleSubmit = (event) => {
+    setIsLoading(true)
+    event.preventDefault()
+    postComment(formData, article_id).then((response)=>{
+        setReturnComment(response)
+        // setCommentAuthor("")
```
Comment (MRombi) [inline fe-nc-news/src/AddComment.jsx]: We should remove comments 🙂

File: fe-nc-news/src/AddComment.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+{if (isLoading===true){
+    return <p>Patience please, your comment is being added...</p>
+}}
+
+return (
+    <>
+    <form onSubmit={handleSubmit}>
+        <label>Username: {commentAuthor}{' '}
+            {/* <input type="text" value={commentAuthor} onChange={(event)=> setCommentAuthor(event.target.value)} required /> */}
```
Comment (MRombi) [inline fe-nc-news/src/AddComment.jsx]: We should remove comments 🙂

**Review by MRombi — APPROVED** [review]
Please, apply the changes I mentioned and you are free to continue 🦾

File: fe-nc-news/src/Topics.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { getTopics, getArticlesByTopic } from "./utils/ApiFunctions";
+import {React, useState, useEffect} from 'react'
```
Comment (Augs0) [inline fe-nc-news/src/Topics.jsx:2]: I don't think we need the React import here

File: fe-nc-news/src/styles/articles.css [hunk 5]
```diff
@@ -31,9 +32,15 @@
...
     bottom: 0;
     }
 
+.newComment {
+    background-color: rgb(244, 114, 62);
+    color: black;
+    border-radius: 10px;
+    padding: 10px;
+}
     /* //position abosloute position relative for link placement set bottom to zero  */
```
Comment (Augs0) [inline fe-nc-news/src/styles/articles.css:46]: Don't forget to remove commented out code

**Review by Augs0 — APPROVED** [review]
Great work, Jay! Have a think about how w could show the user their comment has been posted. It could be as simple as a paragraph that says comment posted or something

**Pull request merged**